module3/homework/passwordCheck/validation.cpp

```diff
@@ -1,2 +1,55 @@
 #include "validation.hpp"
-// TODO: Put implementations here
\ No newline at end of file
+#include <algorithm>
+#include <string>
+
+std::string getErrorMessage(ErrorCode errorNumber) {
+    switch (errorNumber) {
+        case ErrorCode::Ok: {
+            return "OK";
```

Comment:
As I remember Mateusz said under on of the review that it is a good practice to use `break`, even if we are returning something. So maybe we could add them also.

Reply:
I agree, I will change that, return and break is for different cases.